covid19-notebooks/readme.md

```diff
@@ -40,3 +40,15 @@ In this R markdown, we track COVID-19 local (US) and global cases with active, c
 ### "COVID-19 Testing Over Time in the US" notebook:
 
 COVID-19 testing is very crucial to understand the spread of the pandemic. In this python notebook, we have focused our efforts on visualizing the COVID-19 testing data across the U.S. States and Territories from the COVID Tracking Project. The animations will allow you to see how testing data has changed over time.
```

Comment:
Change this line: "U.S. States and Territories" to United States and U.S. Territories